packages/amplication-server/src/core/ai/ai.service.ts

```diff
@@ -18,11 +18,16 @@ import {
 import { KafkaProducerService } from "@amplication/util/nestjs/kafka";
 import { Injectable } from "@nestjs/common";
 import { AiBadFormatResponseError } from "./errors/ai-bad-format-response.error";
+import { AmplicationLogger } from "@amplication/util/nestjs/logging";
+import { BtmManagerService } from "./btm-manager.service";
 
 @Injectable()
 export class AiService {
   constructor(
     private readonly actionService: ActionService,
+    private readonly btmManagerService: BtmManagerService,
+    @Inject(AmplicationLogger)
```

Comment:
Why do we need it with `@Inject` ?

Reply:
This will be handled in the refactoring task https://github.com/amplication/private-issues/issues/127